Cython/Compiler/ExprNodes.py

```diff
@@ -2306,6 +2306,7 @@ def generate_assignment_code(self, rhs, code, overloaded_assignment=False,
             assert entry.type.is_pyobject, "Python global or builtin not a Python object"
             interned_cname = code.intern_identifier(self.entry.name)
             namespace = self.entry.scope.namespace_cname
+            code.putln("{")
```

Comment:
Would be nice to generate the surrounding braces only when needed, i.e. when really assigning in the module scope.